### PR TITLE
update WDC version

### PR DESCRIPTION
**Pull request by rusdevops**
fixed sha1

### CODE REVIEW

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2223
* https://travis-ci.org/ingenue/hunter/builds/267333981?utm_source=github_status&utm_medium=notification

File: cmake/projects/WDC/hunter.cmake [hunk 1]
```diff
@@ -8,6 +8,13 @@ include(hunter_pick_scheme)
 
 hunter_add_version(
     PACKAGE_NAME WDC
+    VERSION "1.1.1"
+    URL "https://github.com/CloudPolis/webdav-client-cpp/archive/v1.1.0.tar.gz"
+    SHA1 ae111eee686c2f7d9091164ba270a6be11f8d4a8
```
Comment (ruslo) [inline cmake/projects/WDC/hunter.cmake]: Should be `cc0923530fd1ce13fe927592bad3999db4f22d5e`.

**Comment by rusdevops** [comment]
I sealed the url.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2225
* https://travis-ci.org/ingenue/hunter/builds/267459767?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2228
* https://travis-ci.org/ingenue/hunter/builds/267506098?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released:
* https://github.com/ruslo/hunter/releases/tag/v0.19.79

**Pull request merged**